src/igvfd/schemas/changelogs/primary_cell.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for *`primary_cell.json`*
 
+### Schema version 12
+
+* Remove `taxa` from `required`.
```

Comment:
I think it would be nice to indicate the property is not only not required but also calculated